src/log_database_proxy_model.cpp

```diff
@@ -42,9 +42,16 @@
 #include <QTimer>
 #include <QSettings>
 #include <swri_console/settings_keys.h>
+#include <QtGlobal>
+#include <algorithm>
+#include <iterator>
```

Comment:
Unless the order is important, it's best to keep headers sorted so that headers from related libraries are grouped together.  (all Qt headers in one block, SwRI headers in another, STL headers in another, etc.)